optuna/samplers/_cmaes.py

```diff
@@ -365,23 +418,59 @@ def _to_optuna_param(distribution: BaseDistribution, cma_param: float) -> Any:
     return cma_param
 
 
-def _initialize_x0(search_space: Dict[str, BaseDistribution]) -> Dict[str, np.ndarray]:
+def _initialize_x0(search_space: Dict[str, BaseDistribution]) -> Dict[str, float]:
     x0 = {}
     for name, distribution in search_space.items():
-        if isinstance(distribution, optuna.distributions.UniformDistribution):
-            x0[name] = np.mean([distribution.high, distribution.low])
-        elif isinstance(distribution, optuna.distributions.DiscreteUniformDistribution):
-            x0[name] = np.mean([distribution.high, distribution.low])
-        elif isinstance(distribution, optuna.distributions.IntUniformDistribution):
-            x0[name] = int(np.mean([distribution.high, distribution.low]))
```

Comment:
I noticed that this returns as external representation. But it seems to return internal representation here because x0 should be `1.5` when given `IntUniformDistribution(low=1, high=3)`.